types/testsuite/suite.go

```diff
@@ -301,6 +301,14 @@ func (suite *TestSuite) GetAccAddress() sdk.AccAddress {
 	return sdk.AccAddress(secp256k1.GenPrivKey().PubKey().Address())
 }
 
+func (suite *TestSuite) FundAccount(bankKeeper bankkeeper.Keeper, ctx sdk.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
```

Comment:
How about using `suite.BankKeeper` instead of bankKeeper parameter?

Reply:
Ah, thank you. It is changed in [af30aae](https://github.com/medibloc/panacea-core/pull/356/commits/af30aae04013616714a318c4a6d609cf58c0420d)